Formula/goredo.rb

```diff
@@ -0,0 +1,29 @@
+class Goredo < Formula
+  desc "Go implementation of djb's redo, a Makefile replacement that sucks less"
+  homepage "http://www.goredo.cypherpunks.ru/"
+  url "http://www.goredo.cypherpunks.ru/download/goredo-0.12.3.tar.zst"
+  version "0.12.3"
```

Comment:
I think specifying the version explicitly is unnecessary here. The version should be parsed automatically from the tarball URL. We only need to specify it when the parser gets confused by extraneous bits of the URL, but that doesn't look like it'll happen here...

Reply:
It seems to get confused, that's why I specified the version explicitly here :shrug:

Reply:
Huh, must be the `.zst` extension. That's fine then...